res/img/lock.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
is this not from the feather-customized design kit?

Reply:
yup. should it be in a different directory?

Reply:
https://github.com/matrix-org/matrix-react-sdk/tree/develop/res/img/feather-customised

Reply:
oh crikey, it is the same exact one, isn't it? for some reason i thought that one was different

Reply:
wait, it's not the same. here's how it looks with the one in `feather-customised` – how should we resolve this?
<img width="311" alt="Screenshot 2020-01-09 at 15 52 10" src="https://user-images.githubusercontent.com/3935731/72082457-17c0a200-32f8-11ea-9ccf-e8af229e0491.png">

Reply:
is this existing one used for anything else?

Reply:
no idea

Reply:
yes, in the settings dialog

perhaps i could bring it in and rename it `lock-solid.svg`?

Reply:
sounds reasonable, needs updating in the Design Kit to prevent future confusion